language_detection/python/test.py

```diff
@@ -1,11 +1,14 @@
 #!/usr/bin/env python
 
-import json, commands
+import json, subprocess
 from jubatus.classifier import client
 from jubatus.common import Datum
 
 while True:
-    buf = raw_input("> ")
+    try:
+        buf = raw_input("> ")
```

Comment:
This line causes error when exiting with Ctrl-C on Python 3.

```
NameError: name 'raw_input' is not defined
```

Maybe it's better to use function reference like:

```
try:
  # for Python 2
  input_func = raw_input
except NameError:
  # for Python 3
  input_func = input
  pass
```